debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/j9/gc/GCObjectHeapIteratorAddressOrderedList_V1.java

```diff
@@ -150,7 +182,7 @@ private U8Pointer adjustedToRange(U8Pointer ptr, U8Pointer base, U8Pointer top)
 		}
 		return result;
 	}
-
+	
```

Comment:
Please don't add trailing whitespace (in fact, I support removing it elsewhere).